Source/PercentageGradient.swift

```diff
@@ -7,19 +7,55 @@ public enum GradientDirection {
 
 }
 
+public struct GradientPoints {
+    
+    public var start: CGPoint
+    public var end: CGPoint
+    
+}
+
+public enum GradientAngle {
```

Comment:
What do you think of a `.custom(start: end:)` option as well? That way we can open up the ability to make gradients on any angle the developer may want to implement, rather than just along the common cases we've covered.

Reply:
Can totally do that, it will bring more consistency to the whole thing